tmt-app/src/Components/GuideSidebar.js

```diff
@@ -1,26 +1,52 @@
 import React from "react";
 import { FlexContainer, PrimaryBtn } from "../Styling/CustomStyling.js";
-import { Box, Typography } from "@mui/material";
+import { Box, Typography, Link } from "@mui/material";
+import { Link as RouterLink } from "react-router-dom";
 
 function GuideSidebar({ toggleButton }) {
   return (
     <FlexContainer sx={{ flexDirection: "column" }}>
-      <Box sx={{ width: "60%", textAlign: "center" }}>
+      <Box
+        sx={{
+          width: {
+            xs: "175px",
+            sm: "225px",
+            xl: "275px",
+          },
+          textAlign: "center",
+        }}
+      >
         <Box>
           <Typography variant="h1">Major Guide</Typography>
         </Box>
         <Box>
           <Typography variant="h3">Explore majors, careers, & more</Typography>
         </Box>
       </Box>
-      <PrimaryBtn
-        disabled={toggleButton}
-        sx={{
-          backgroundColor: toggleButton ? "InactiveGrey2.main" : "Matcha.main",
-        }}
+      <Link
+        sx={{ color: "Boba.main" }}
+        underline="none"
+        aria-label="Learn more about other different majors"
+        component={RouterLink}
+        to="/quiz/1"
       >
-        <Typography variant="h3">Take the Quiz</Typography>
-      </PrimaryBtn>
+        <PrimaryBtn
+          disabled={toggleButton}
+          sx={{
+            backgroundColor: toggleButton
+              ? "InactiveGrey2.main"
+              : "Matcha.main",
+            "&.MuiButtonBase-root": {
+              padding: {
+                xs: "1rem 2.5rem",
+                xl: "1rem 3.25rem",
+              },
+            },
+          }}
+        >
+          <Typography variant="h4">Take the Quiz</Typography>
+        </PrimaryBtn>
+      </Link>
```

Comment:
Can this be extracted to a component and reused from the home page? 

Non blocking but this would be more of a WET solution knowing we have the exact same button to the same page in another component

Reply:
It could be possible but this button have some custom styling and props that are different from the home page button.

Reply:
Sounds like you're describing props ;)